test/test_plugin.py

```diff
@@ -126,3 +126,84 @@ def foo():
     doc2 = Document(DOC_URI, ws2, DOC_SOURCE)
     diags = plugin.pylsp_lint(ws2._config, ws2, doc2, is_saved=False)
     assert len(diags) == 0
+
+
+def test_apply_overrides():
+    assert list(plugin.apply_overrides(["1", "2"], [])) == []
+    assert list(plugin.apply_overrides(["1", "2"], ["a"])) == ["a"]
+    assert list(plugin.apply_overrides(["1", "2"], ["a", True])) == ["a", "1", "2"]
+    assert list(plugin.apply_overrides(["1", "2"], [True, "a"])) == ["1", "2", "a"]
+    assert list(plugin.apply_overrides(["1"], ["a", True, "b"])) == ["a", "1", "b"]
+
+
+def test_option_overrides(tmpdir, diag_mp, workspace):
+    import sys
+    from textwrap import dedent
+
+    sentinel = tmpdir / "ran"
+
+    source = dedent(
+        """\
+        #!{}
+        import os, sys, pathlib
+        pathlib.Path({!r}).touch()
+        os.execv({!r}, sys.argv)
+        """
+    ).format(sys.executable, str(sentinel), sys.executable)
+
+    wrapper = tmpdir / "bin/wrapper"
+    wrapper.write(source, ensure=True)
+    wrapper.chmod(0o700)
```

Comment:
Maybe use the constant instead for better readability

Reply:
Hmm... the browser UI is now showing me a different snippet than the one that appeared in the notification email containing your initial comments. I originally thought you were talking about the use of dedent, so that's what I changed. But now I'm not so sure.

Reply:
I am find with the dedent, i was talking about the chmod, the argument can to my knowledge also be expressed with constants which imho is a lot more readable.

Reply:
Ah yes, makes sense. Fixed now (in another fix-up). Let me know if you'd like me to squash them down. Thanks.